react/features/conference/components/web/Conference.js

```diff
@@ -231,7 +232,17 @@ class Conference extends AbstractConference<Props, *> {
                         }
                     </div>
 
-                    { _showPrejoin || _showLobby || <Toolbox /> }
+                    { _showPrejoin || _showLobby || (
+                        <>
```

Comment:
Is there a reason why role = 'heading' and not 'toolbar'?
Trying to understand this. 
For example, I understand why we applied it on ChatHeader, but here it's a bit confusing.

Reply:
Hey, finally back :upside_down_face: 

The "toolbar" role would be semantically good for sure, but in a practical way, this role is not used by most (if not all?) screen readers.

In the end it's better for the user that we use the "heading" role saying "Toolbar":

- the screen reader user understands it's the toolbar thanks to the "Toolbar" content,
- and he can use the headings-related keyboard shortcuts to navigate to and from the toolbar. Those kind of shortcuts are a common navigation pattern when using a screen reader. He wouldn't be able to do this if we used the "toolbar" role